packages/governance/src/types.js

```diff
@@ -490,11 +475,17 @@
  */
 
 /**
- * @typedef {Object} LimitedCreatorFacet
+ * @typedef {{
+ * }} GovernableCreatorFacet
```

Comment:
I forget, is there really no requirement on a creator facet to be governed?

if so, why name it?

Reply:
Everything required is provided. I'll drop it.  I don't think we'll be adding requirements back.